### PR TITLE
Fix display of specific user details when clicked

### PR DESCRIPTION
**Pull request by RamuKammagani-osi**

### CODE REVIEW

File: app/javascript/sagas/getDetailsSaga.test.js [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
+  getUserId,
+  currentPathname,
+} from './getDetailsSaga';
 import * as actionTypes from '../actions/actionTypes';
 import { takeLatest, call, put } from 'redux-saga/effects';
 
-const id = 'id';
-UserService.fetchUserDetails(id);
+// const id = 'id';
+// UserService.fetchUserDetails(id);
```
Comment (kelfink) [inline app/javascript/sagas/getDetailsSaga.test.js]: remove these lines?

File: docker-compose.yml [hunk 2]
```diff
@@ -43,4 +53,4 @@ services:
       - PERRY_API_BASE_URL
       - COUNTY_ADMIN_WEB_BASE_URL
       - DASHBOARD_URL
-      - RAILS_RELATIVE_URL_ROOT
+      - RAILS_RELATIVE_URL_ROOT
```
Comment (kelfink) [inline docker-compose.yml]: sad no new line at end of file

File: app/javascript/sagas/getDetailsSaga.test.js (hunk 1 above)
Comment (RamuKammagani-osi) [reply app/javascript/sagas/getDetailsSaga.test.js]: Will do, Thanks

File: docker-compose.yml (hunk 2 above)
Comment (RamuKammagani-osi) [reply docker-compose.yml]: Will do, Thanks

**Pull request merged**